app/react/src/server/framework-preset-react-docgen.ts

```diff
@@ -27,24 +30,13 @@ export function babel(config: TransformOptions, { typescriptOptions }: Storybook
 
 export function webpackFinal(config: Configuration, { typescriptOptions }: StorybookOptions) {
   const { reactDocgen, reactDocgenTypescriptOptions } = typescriptOptions;
-  if (reactDocgen !== 'react-docgen-typescript') return config;
+
+  if (reactDocgen !== 'react-docgen-typescript') {
```

Comment:
Why was this changed?

Reply:
I personally prefer the *new* code

Reply:
I was more pointing out that this change isn't related to any of the work. I think it's good practice to try to only change things related to the task at hand, unless something is being improved. But if you think the change is meaningful, then I'm OK with that.